gitea/README.md

```diff
@@ -0,0 +1,75 @@
+# Gitea Integration
+
+## Overview
+
+[Gitea][1] is a community managed lightweight code hosting solution written in Go.
+
+This integration monitors Gitea instances through the Datadog [Agent][2].
+
+## Setup
+
+### Prerequisite
+
+Gitea doesn't expose its internal metrics by default. You need to enable the built-in HTTP server that exposes the metrics endpoint in your `app.ini` configuration file.
+
+```ini
+[metrics]
+ENABLED = true
+```
+
+See the official [documentation][3] for more information.
+
+### Installation
+
+The Gitea integration is not included in the [Datadog Agent][4] package by default, so you need to install it.
+
+For Agent v7.36+, follow the instructions below to install the Gitea check on your host. See [Use Community Integrations][5] to install with the Docker Agent or earlier versions of the Agent.
+
+1. Run the following command to install the Agent integration:
+
+```shell
+datadog-agent integration install -t datadog-gitea==<INTEGRATION_VERSION>
+```
+
+2. Configure your integration similar to core [integrations][6].
```

Comment:
Not sure if `core` is the best word here -- the linked page mentions `Agent-based`, `Authentication`, and `Library` integrations, and I'm assuming this is referring to Agent-based. If so, it may be helpful to clarify this for users here.

Reply:
You're right. Updated! 

To be honest, I stole this section from another integrations. I might update this other integration as well in another PR?

Reply:
Thanks for confirming! And by all means, please feel free to clarify this in other pages as well. You can also let me know any such pages and I can create a card internally to update them.